src/Illuminate/Console/Scheduling/ScheduleListCommand.php

```diff
@@ -37,8 +37,9 @@ public function handle(Schedule $schedule)
                 $event->expression,
                 $event->description,
                 (new CronExpression($event->expression))
-                            ->getNextRunDate(Carbon::now())
-                            ->setTimezone($this->option('timezone', config('app.timezone'))),
+                            ->getNextRunDate(Carbon::now()->setTimezone($event->timezone))
+                            ->setTimezone($this->option('timezone', config('app.timezone')))
```

Comment:
Can't have this foundation config function call.

Reply:
Do you mean `config('app.timezone')`? It was originally there.

Pros: all timezones are based on Laravel's `.env`
Cons: the timezone displayed on output may not be the same as coded, i.e. coded as 0:00 GMT+0, but displayed as 1:00 GMT+1.

Reply:
@GrahamCampbell for command's it's okay since they only work with the base foundation anyway.